src/Catel.Test/Catel.Test.Shared/Collections/FastObservableCollectionFacts.cs

```diff
@@ -144,5 +145,141 @@ public void ReturnsSingleElementUsingLinq()
                 Assert.AreEqual(42, allInts);
             }
         }
+
+        [TestFixture]
+        public class TheSuspensionModeMethod
+        {
+            [TestCase]
+            public void AddingItemsInAddingMode()
+            {
+                var counter = 0;
+                var eventArgs = (NotifyCollectionChangedEventArgs)null;
+
+                var fastCollection = new FastObservableCollection<int>();
+                fastCollection.AutomaticallyDispatchChangeNotifications = false;
+                fastCollection.CollectionChanged += (sender, e) =>
+                    {
+                        counter++;
+                        eventArgs = e;
+                    };
+
+                using (fastCollection.SuspendChangeNotifications(SuspensionMode.Adding))
+                {
+                    fastCollection.Add(1);
+                    fastCollection.Add(2);
+                    fastCollection.Add(3);
+                    fastCollection.Add(4);
+                    fastCollection.Add(5);
+                }
+
+                Assert.AreEqual(1, counter);
+                Assert.AreEqual(NotifyCollectionChangedAction.Add, eventArgs.Action);
+                CollectionAssert.AreEqual(eventArgs.NewItems, new[] { 1, 2, 3, 4, 5 });
+            }
+
+            [TestCase]
+            public void MultipleActionsWithoutSuspendingNotifications()
```

Comment:
ok, let's keep it for now, but I don't think it's that complex. Let's keep this for phase 2 ;-)